package.json

```diff
@@ -76,22 +76,22 @@
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "eslint-plugin-sonarjs": "^0.19.0",
     "eslint-plugin-storybook": "^0.6.11",
-    "eslint-plugin-svelte3": "^4.0.0",
+    "eslint-plugin-svelte": "^2.31.0",
     "eslint-plugin-tailwindcss": "^3.11.0",
     "eslint-plugin-unicorn": "^46.0.0",
     "monaco-editor": "^0.37.1",
     "postcss": "^8.4.23",
     "prettier": "^2.8.8",
-    "prettier-plugin-svelte": "^2.10.0",
+    "prettier-plugin-svelte": "^2.10.1",
     "prismjs": "^1.29.0",
     "react": "^18.2.0",
     "remark-gfm": "^3.0.1",
     "storybook": "^7.0.7",
-    "svelte": "^3.58.0",
-    "svelte-check": "^3.2.0",
+    "svelte": "^4.0.0",
+    "svelte-check": "^3.4.3",
     "svelte-preprocess": "^5.0.3",
     "tailwindcss": "^3.3.1",
-    "typescript": "^5.0.4",
```

Comment:
y go back?

Reply:
I believe this was changed by the svelte upgrade script. I upgraded to the latest.